core/components/atoms/tooltip/Tooltip.tsx

```diff
@@ -81,18 +74,14 @@ export class Tooltip extends React.Component<TooltipProps, IState> {
       tooltip,
       children,
       className,
+      triggerClass,
       ...props
     } = this.props;
 
-    const classes = classNames({
-      ['Tooltip']: true
-    }, className);
-
     const tooltipWrapper = (
       <div
-        className={classes}
+        className="Tooltip ellipsis"
```

Comment:
Why tooltip has an `ellipsis` class.

Reply:
If the length of string is more than max-width of tooltip, that particular string will have ellipsis and other strings will be moved to next line.